ocpmodels/trainers/base_trainer.py

```diff
@@ -76,9 +75,11 @@ def __init__(
                 "print_every": print_every,
                 "seed": seed,
                 "timestamp": timestamp,
-                "checkpoint_dir": run_dir / "checkpoints" / timestamp,
-                "results_dir": run_dir / "results" / timestamp,
-                "logs_dir": run_dir / "logs" / logger / timestamp,
+                "checkpoint_dir": os.path.join(
```

Comment:
You can also write this as `str(run_dir / "checkpoints" / timestamp)` if you want.